browser/app/js/components/BrowserDropdown.js

```diff
@@ -18,39 +18,40 @@ import React from 'react'
 import connect from 'react-redux/lib/components/connect'
 import Dropdown from 'react-bootstrap/lib/Dropdown'
 
+
 let BrowserDropdown = ({fullScreenFunc, aboutFunc, settingsFunc, logoutFunc}) => {
   return (
-    <li>
-      <Dropdown pullRight id="top-right-menu">
+    <nav className="top-links">
+      <Dropdown pullRight id="dropdown-top-links">
         <Dropdown.Toggle noCaret>
-          <i className="fa fa-reorder"></i>
+          <span href=""><i className="zmdi zmdi-more-vert" /></span>
         </Dropdown.Toggle>
         <Dropdown.Menu className="dropdown-menu-right">
           <li>
-            <a target="_blank" href="https://github.com/minio/minio">Github <i className="fa fa-github"></i></a>
+            <a onClick={ aboutFunc }>About <i className="zmdi zmdi-info" /></a>
           </li>
           <li>
-            <a href="" onClick={ fullScreenFunc }>Fullscreen <i className="fa fa-expand"></i></a>
+            <a onClick={ settingsFunc }>Settings <i className="zmdi zmdi-settings" /></a>
```

Comment:
Right now the settings page just shows the change password modal, with the title of "CHANGE PASSWORD". Would it be better to name this dropdown link "Change Password"? Or perhaps we should have the modal named "SETTINGS" and a sub-header named "Change Password" or something? I think there's a discrepancy between the two.

Edit: also these icons are just one or two pixels below the text on Firefox. It looks OK on chrome.
![screen shot 2017-07-31 at 5 14 11 pm](https://user-images.githubusercontent.com/3038254/28803771-cb57912e-7613-11e7-95a4-1062b7a5ee6f.png)